soroban-sdk/src/_migrating/v23_token_transfer.rs

```diff
@@ -55,9 +51,13 @@
 //!    let to = muxed_to.address();
 //!    // Token-specific implementation of balance movement (same as before).
 //!    token_impl::move_balance(&env, &from, &to, amount);
-//!    // Publish an appropriate transfer event that includes the muxed ID
-//!    // when it's non-None.
-//!    token::publish_transfer_event(&env, &from, &muxed_to, amount);
+//!    // Publish the transfer event.
+//!    token::Transfer {
```

Comment:
My main concern about this is that this would lead to confusion when emitting transfer events not from the transfer. Though probably the helper function would lead to the same sort of confusion, so maybe it's fine.